source/sdk/lang/Memory.ooc

```diff
@@ -1,53 +1,20 @@
 version(windows) { include malloc }
 else { include alloca }
 
-version(!gc) {
-    // GC_MALLOC zeroes the memory, so in the non-gc version, we prefer to use calloc
-    // to the expense of some performance. Don't use malloc instead - existing ooc code
-    // is written assuming that gc_malloc zeroes the memory.
-    gc_malloc: func (size: SizeT) -> Pointer {
-        gc_calloc(1, size)
-    }
-    gc_malloc_atomic: extern(malloc) func (size: SizeT) -> Pointer
-    gc_strdup: extern(strdup) func (str: CString) -> CString
-    gc_realloc: extern(realloc) func (ptr: Pointer, size: SizeT) -> Pointer
-    gc_calloc: extern(calloc) func (nmemb: SizeT, size: SizeT) -> Pointer
-    gc_free: extern(free) func (ptr: Pointer)
-}
-
-version(gc) {
-    include gc/gc | (GC_THREADS)
-    // to get the GC_pthread_* prototypes
-    include gc/gc_pthread_redirects | (GC_NO_THREAD_REDIRECTS)
-
-    gc_malloc_for_generics: func (size: SizeT) -> Pointer {
-        gc_malloc(size)
-    }
-
-    gc_malloc: extern(GC_malloc) func (size: SizeT) -> Pointer
-    gc_malloc_atomic: extern(GC_malloc_atomic) func (size: SizeT) -> Pointer
-    gc_malloc_uncollectable: extern(GC_malloc_uncollectable) func (size: SizeT) -> Pointer
-    gc_strdup: extern(GC_strdup) func (str: CString) -> CString
-    gc_realloc: extern(GC_realloc) func (ptr: Pointer, size: SizeT) -> Pointer
-    gc_calloc: func (nmemb: SizeT, size: SizeT) -> Pointer {
-        gc_malloc(nmemb * size)
-    }
-    gc_free: extern(GC_free) func (ptr: Pointer)
-
-    gc_register_finalizer: extern(GC_register_finalizer) func (ptr: Pointer, finalizer: Pointer, userdata: Pointer, wtf1: Pointer, wtf2: Pointer)
-    gc_register_finalizer_ignore_self: extern(GC_register_finalizer_ignore_self) func (ptr: Pointer, finalizer: Pointer, userdata: Pointer, wtf1: Pointer, wtf2: Pointer)
-    gc_register_finalizer_no_order: extern(GC_register_finalizer_no_order) func (ptr: Pointer, finalizer: Pointer, userdata: Pointer, wtf1: Pointer, wtf2: Pointer)
-
-    GC_add_roots: extern func (Pointer, Pointer)
-    GC_remove_roots: extern func (Pointer, Pointer)
-    GC_stackbottom: extern Pointer
-}
-
-// memory management
+// GC_MALLOC zeroes the memory, so in the non-gc version, we prefer to use calloc
+// to the expense of some performance. Don't use malloc instead - existing ooc code
+// is written assuming that gc_malloc zeroes the memory.
+gc_malloc: func (size: SizeT) -> Pointer { gc_calloc(1, size) }
+gc_malloc_atomic: extern(malloc) func (size: SizeT) -> Pointer
+gc_strdup: extern(strdup) func (str: CString) -> CString
+gc_realloc: extern(realloc) func (ptr: Pointer, size: SizeT) -> Pointer
+gc_calloc: extern(calloc) func (nmemb: SizeT, size: SizeT) -> Pointer
+gc_free: extern(free) func (ptr: Pointer)
+
+//TODO Remove the above declarations once they're no longer used - only use the following:
+malloc: extern(malloc) func (size: SizeT) -> Pointer
+calloc: extern(calloc) func (nmemb: SizeT, size: SizeT) -> Pointer
```

Comment:
```ooc
malloc: extern func (SizeT) -> Pointer
calloc: extern func (SizeT, SizeT) -> Pointer
```
would do just as nicely. If you want parameter names, please use something more descriptive than `nmemb`.